.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,6 @@
+We are always happy to receive pull requests.
```

Comment:
How do we decide whether a PR gets merged? Bugfix vs. added features? Relevance to what people within the mlr team are doing? Code quality? Maintainability? Willingness of the submitter to support their part of the code? Unspoken feelings of obligation when someone obviously puts lots of effort into his PR? Hope to get someone invested in mlr to make him post higher quality PRs in the future? (I actually don't know, but maybe we should make some of these explicit here.)